garak/generators/litellm.py

```diff
@@ -122,15 +122,18 @@ def __init__(self, name: str = "", generations: int = 10, config_root=_config):
 
     @backoff.on_exception(backoff.fibo, litellm.exceptions.APIError, max_value=70)
     def _call_model(
-        self, prompt: str, generations_this_call: int = 1
-    ) -> List[Union[str, None]]:
-        if isinstance(prompt, str):
-            prompt = [{"role": "user", "content": prompt}]
+        self, prompt: Conversation, generations_this_call: int = 1
+    ) -> List[Union[Message, None]]:
+        if isinstance(prompt, Conversation):
+            litellm_prompt = []
+            for turn in prompt.turns:
+                litellm_prompt.append({"role": turn.role, "content": turn.content.text})
         elif isinstance(prompt, list):
-            prompt = prompt
+            # should we maintain support for list here?
+            litellm_prompt = prompt
         else:
             msg = (
-                f"Expected a list of dicts for LiteLLM model {self.name}, but got {type(prompt)} instead. "
+                f"Expected list or Conversation for LiteLLM model {self.name}, but got {type(prompt)} instead. "
```

Comment:
```suggestion
                f"Expected list of Conversation for LiteLLM model {self.name}, but got {type(prompt)} instead. "
```

Reply:
as noted in line 132 comment, this generator expects `List|Conversation` in runtime testing.  Happy to make this:
```suggestion
                f"Expected `Conversation` for LiteLLM model {self.name}, but got {type(prompt)} instead. "
```
If we are removing that support, however `_call_model()` should not accept a `"list of Conversation"`.